storybook/storybook-react/config/story.css

```diff
@@ -0,0 +1,5 @@
+/* Workaround to mimic dark background for dialog Canvas blocks */
+.amsterdam-dialog-story {
+  background-color: #333 !important;
```

Comment:
Is this used? It seems like it's either `::backdrop` or `parameters.backgrounds.default: 'dark'` that set the background color.

Reply:
Ah, I missed that one